src/deprecation.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2017 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef ZCASH_DEPRECATION_H
+#define ZCASH_DEPRECATION_H
+
+// Deprecation policy is 4th third-Tuesday after a release
+static const int APPROX_RELEASE_HEIGHT = 115000;
```

Comment:
I did read the whole PR. My rationale was someone reading this code who doesn't know our release process would know how we calculated the number. It's not important enough to block the PR, and the comment on the preceding line should make it clear enough.